apps/src/code-studio/initApp/loadApp.js

```diff
@@ -425,6 +425,18 @@ window.apps = {
       } else {
         callback({});
       }
+    },
+    prepareForRemix: function () {
+      let prepareForRemix = getAppOptions().prepareForRemix;
+      return new Promise((resolve, reject) => {
+        if (prepareForRemix) {
+          prepareForRemix().then(() => {
+            resolve();
+          });
+        } else {
+          resolve();
+        }
```

Comment:
I like that. I was hoping there was a way to return an insta-resolved promise, but didn't look it up 😄 
The ES6 destructuring is cool, but strange. I can't quite parse that - but I'm sure I'll get used to it!
And avoiding creating an extra promise makes lots of sense for the case where it is available on `getAppOptions()`